Sources/PrimerSDK/Classes/Core/Primer/Primer.swift

```diff
@@ -174,25 +176,19 @@ public class Primer {
                     .buckarooSofort,
                     .coinbase,
                     .hoolah,
+                    .klarna,
```

Comment:
do we need `.klarna` among these payment method cases? asking as I've seen `.klarna` below (in the `else` condition) as well